### PR TITLE
Respect TMPDIR

### PR DESCRIPTION
**Pull request by thesamesam**
Signed-off-by: Sam James <sam@gentoo.org>

### CODE REVIEW

File: wgetpaste [hunk 1]
```diff
@@ -1049,7 +1049,7 @@ if [[ $SERVICE == tinyurl ]]; then
 else
 	# create temp file (wget is much more reliable reading
 	# large input via --post-file rather than --post-data)
-	[[ -f $TMPF ]] || TMPF=$(mktemp /tmp/wgetpaste.XXXXXX)
+	[[ -f $TMPF ]] || TMPF=$(mktemp -q -d -tmpdir wgetpaste.XXXXXX)
```
Comment (xxc3nsoredxx) [inline wgetpaste:1052]: Ditto, and I think you're missing a `-` in the `--tmpdir`

File: test/test.sh [hunk 2]
```diff
@@ -1,13 +1,13 @@
-#! /bin/bash
+#!/usr/bin/env bash
 
 # wgetpaste test script
 # Exit code: number of mismatched downloads or 1 for general failure
-# Copyright (C) 2021  xxc3nsoredxx
+# Copyright (C) 2022 Oskari Pirhonen <xxc3ncoredxx@gmail.com>
```
Comment (xxc3nsoredxx) [inline test/test.sh:5]: I'm ok with this, but shouldn't it be "2021-2022" since that's when it was first created?

**Review by xxc3nsoredxx — CHANGES_REQUESTED** [review]

**Comment by xxc3nsoredxx** [comment]
lgtm

**Comment by heirecka** [comment]
Thanks!

**Pull request merged**